File: Math/find_n_unique_integers_sum_up_to_zero.py
class Solution:
    def sumZero(self, n: int) -> list[int]:
        start = n // 2
        if not n & 1:
            return [x for x in range(-start, start+1) if x != 0]
        else:
            return [x for x in range(-start, start+1)]

File: Math/CPP/find_n_unique_integers_sum_up_to_zero.cpp
#include <vector>
using namespace std;

class Solution {
public:
    vector<int> sumZero(int n) {
        int mid = n / 2;
        vector<int> res;
        for (int i = -mid; i < mid + 1; i++) {
            if (i == 0 && n % 2 == 0) continue;
            else res.push_back(i);
        }
        return res;
    }
};